superset/views/base.py

```diff
@@ -392,7 +393,7 @@ def check_ownership(obj, raise_if_false=True):
     orig_obj = scoped_session.query(obj.__class__).filter_by(id=obj.id).first()
 
     # Making a list of owners that works across ORM models
-    owners = []
+    owners: User = []
```

Comment:
Should this be `List[User]` ?

Reply:
BTW the reasons these issues didn't fire is merely because the enforcement is on the `superset.charts` module. It simply requires the calls to be typed.